scipy/_lib/tests/test__util.py

```diff
@@ -105,11 +107,13 @@ def test_mapwrapper_serial():
         p = MapWrapper(0)
 
 
+@pytest.mark.filterwarnings("ignore:.*JAX is multithreaded.*:RuntimeWarning")
```

Comment:
we should probably filter these package-wide to avoid clutter, especially given the 'let's not go out of our way to support a proprietary library' argument. Ideally, including JAX here is strictly helpful to us on almost the entire diff.